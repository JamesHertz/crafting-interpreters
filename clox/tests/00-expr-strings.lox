print "Hello, world!";
print "Hello, " + "world!";

print "hello " + "number " + 10;
print false + " hello";

print "hello" == "hello";
print true + "" == "true";

print true == "false";
print true != "false";

print "hi" == "hello";
print "hi" != "hello";
