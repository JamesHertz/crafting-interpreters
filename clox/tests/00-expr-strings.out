Hello, world!
Hello, world!
hello number 10
false hello
true
true
false
true
false
true
